### ISSUE TITLE
page request: flutter test

### ISSUE THREAD

**Issue by phrohdoh** [tldr-pages/tldr#4190]
https://flutter.dev/docs/cookbook/testing/unit/introduction#run-tests-in-a-terminal

```
$ flutter test --help
Run Flutter unit tests for the current project.

Usage: flutter test [arguments]
-h, --help                            Print this usage information.
    --[no-]pub                        Whether to run "flutter pub get" before executing this command.
                                      (defaults to on)
    --name=<regexp>                   A regular expression matching substrings of the names of tests to run.
    --plain-name=<substring>          A plain-text substring of the names of tests to run.
    --start-paused                    Start in a paused mode and wait for a debugger to connect.
                                      You must specify a single test file to run, explicitly.
                                      Instructions for connecting with a debugger are printed to the console once the test has started.
    --coverage                        Whether to collect coverage information.
    --merge-coverage                  Whether to merge coverage data with "coverage/lcov.base.info".
                                      Implies collecting coverage data. (Requires lcov)
    --coverage-path                   Where to store coverage information (if coverage is enabled).
                                      (defaults to "coverage/lcov.info")
    --update-goldens                  Whether matchesGoldenFile() calls within your test methods should update the golden files rather than test for an existing match.
-j, --concurrency=<jobs>              The number of concurrent test processes to run.
                                      (defaults to "10")
    --[no-]test-assets                Whether to build the assets bundle for testing.
                                      Consider using --no-test-assets if assets are not required.
                                      (defaults to on)
    --platform                        The platform to run the unit tests on. Defaults to "tester".
                                      [tester (default), chrome]
    --test-randomize-ordering-seed    The seed to randomize the execution order of test cases.
                                      Must be a 32bit unsigned integer or "random".
                                      If "random", pick a random seed to use.
                                      If not passed, do not randomize test case execution order.
    --[no-]track-widget-creation      Track widget creation locations. This enables features such as the widget inspector. This parameter is only functional in debug mode (i.e. when compiling JIT, not AOT).
                                      (defaults to on)

Run "flutter help" to see global options.
```

For posterity, the version of `flutter` that generated the above is as follows:

```
$ flutter --version
Flutter 1.17.5 • channel unknown • unknown source
Framework • revision 8af6b2f038 (2 weeks ago) • 2020-06-30 12:53:55 -0700
Engine • revision ee76268252
Tools • Dart 2.8.4
```

**Comment by sbrl** [comment]
Thanks for requesting this page! Do you want to have a go yourself?

Pages are best created by people who understand the command they are about.

If you do, take a quick read of [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md), which guides you through the process of creating a new page here. Then, just simply send a pull request to this repository - and if you need any assistance, just ask :-)

**Comment by bl-ue** [comment]
Thank you @ev-john! Please link the issue number in the PR description so that GitHub automatically closes this issue when the PR is merged.

Closed by PR #4625
